config/eck.yaml

```diff
@@ -14,5 +15,7 @@ disable-telemetry: false
 distribution-channel: image
 validate-storage-class: true
 enable-webhook: false
+operator-namespace: elastic-system
```

Comment:
Default is empty string?

Reply:
This is coming from `manifest-gen` which defaults to using the 'elastic-system' namespace when generating yaml from the helm charts. 
https://github.com/elastic/cloud-on-k8s/blob/20f11e756989d9811d3f22f638949a81409a4ef6/hack/manifest-gen/main.go#L121
And with the addition of `operator-namespace: {{ .Release.Namespace }}`, this is why it's now defaulting to setting this in the default configuration.

If preferred, I can just remove this from the template, as it really isn't getting us anything.